server/tls/tlscertificate_retriever_linux.go

```diff
@@ -0,0 +1,112 @@
+// +build linux
```

Comment:
If the file is suffixed with `linux`, then the build tag is not needed

Reply:
Thanks, I removed it and I added a comment to this to make it easier for future readers of these files.